src/marqo/tensor_search/tensor_search.py

```diff
@@ -2024,7 +2085,7 @@ def vectorise_multimodal_combination_field_unstructured(field: str,
 def vectorise_multimodal_combination_field_structured(
         field: str, multimodal_object: Dict[str, dict], doc: dict, doc_index: int,
         doc_id: str, device: str, marqo_index: StructuredMarqoIndex, image_repo, field_map: dict,
-        model_auth: Optional[ModelAuth] = None
+        model_auth: Optional[ModelAuth] = None, text_chunk_prefix: str = ""
```

Comment:
To keep the defaults consistent, the default for `text_chunk_prefix` should also be `None`

Reply:
revised!